README.rst

```diff
@@ -2116,6 +2116,101 @@ Remember to change ``geo_views`` location in ``urls.py`` in point 11 for extendi
 
 For more information about django views, please refer to the `views section in the django documentation <https://docs.djangoproject.com/en/dev/topics/http/views/>`_.
 
+Registering / Unregistering Commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OpenWISP Controller provides registering and unregistering commands
+through utility functions ``openwisp_controller.connection.commands.register_command``
+and ``openwisp_notifications.types.unregister_notification_type``.
+Using these functions you can register or unregister commands from your code.
+
+``register_command``
+^^^^^^^^^^^^^^^^^^^^
+
++--------------------+----------------------------------------------------------------------+
+| Parameter          | Description                                                          |
++--------------------+----------------------------------------------------------------------+
+| ``command_name``   | A ``str`` defining name of the command.                              |
++--------------------+----------------------------------------------------------------------+
+| ``command_config`` | A ``dict`` defining configuration of the command.                    |
+|                    | It should contain three keys as described below:                     |
+|                    |                                                                      |
+|                    | +--------------+---------------------------------------------------+ |
+|                    | | ``label``    | A ``str`` defining label for the command used     | |
+|                    | |              | internally by Django                              | |
+|                    | +--------------+---------------------------------------------------+ |
+|                    | | ``schema``   | A ``dict`` defining  JSONSchema for inputs of     | |
+|                    | |              | command. It is used to generate input form i      | |
+|                    | |              | required.                                         | |
+|                    | +--------------+---------------------------------------------------+ |
+|                    | | ``callable`` | A ``callable`` or ``str`` defining dotted path to | |
+|                    | |              | a callable. It should return the command to be    | |
+|                    | |              | executed. Inputs of the command are passed as     | |
+|                    | |              | arguments to this callable.                       | |
+|                    | +--------------+---------------------------------------------------+ |
++--------------------+----------------------------------------------------------------------+
+
+**Note:** It will raise ``ImproperlyConfigured`` exception if a command is already
+registered with the same name.
+
```

Comment:
can you add a screenshot with the resulting web interface for this hypothetical ping command to make it super obvious to readers what we are doing in the example?